Generating New Applications.md

```diff
@@ -1,23 +1,26 @@
-## PreCreation Checkup (Team Lead)
+## PreCreation checklist (Devops)
 
 Before a developer creates a new application, a couple of 3rd party services need to be setup.
 
-* **Bitbucket repository** - Create a repository for the project with template: 'web_{app_name}'
+* **Bitbucket repository** - Create a repository for the project with template: [web-{app_name}](https://github.com/infinum/internal-guides/blob/master/How%20to%20name%20your%20repository.md)
 * **Bugsnag** - Create a new project on bugsnag and add developers to it
-* **Semaphore** - Create a new project, link it to bitbucket, integrate it with slack
+* **Semaphore**
+  - Create a new project, link it to bitbucket, integrate it with slack
+  - Create new servers for staging and production to be automatically deployed
 * **Codeclimate** - Create a new project, link it to bitbucket, integrate it with semaphore and slack
 * **Vault** - Create a new github team, create new policies for that team, server the application will run and a app_id for semaphore
 
-## Generating New Rails Application
+* RDS databases for staging and production
+* Server nginx conf
+* Add DNS records
 
-99% of our work is based on the Rails framework. We have some standard configurations and gems that are basically in every Rails application and to make our life easier we made a gem called [Initiate](https://github.com/infinum/initiate) to generate new Rails applications with those standards.
+## Generating New Rails Application
 
-Please check Initiate's [README](https://github.com/infinum/initiate#initiate-) for more information.
+99% of our work is based on the Rails framework. We have some standard configurations and gems that are basically in every Rails application and to make our life easier we made a [default template](https://github.com/infinum/default_rails_template/) to generate new Rails applications with those standards.
 
-**Use Initiate after you've done the Architecture Review.**
+Use `rails new myapp --database=postgresql -T -B -m https://raw.githubusercontent.com/infinum/default_rails_template/master/template.rb` to create a new application.
```

Comment:
I would remove this since it's already written in the repo's readme. If we update it (and we probably will) we will have to change it in two places.